web/components/home/Home.js

```diff
@@ -152,7 +152,17 @@ class Home extends React.Component {
 
     if (!isFeatureFlagsLoaded) return null;
 
-    const { application: { enabled: isApplicationEnabled } } = featureFlagsData;
+    const {
+      application: {
```

Comment:
Minor: this would be better if it were `hacker_application` with `isHackerApplicationEnabled`